### PR TITLE
Add skeleton to use components + prerendering

### PR DESCRIPTION
**Pull request by rynowak**
This adds the skeleton needed for components + prerendring development
in the MVC sandbox. We don't currently have a sample app for quick and
dirty testing.

### CODE REVIEW

**Comment by rynowak** [comment]
@aspnet/brt

**Comment by rynowak** [comment]
@javiercn

File: src/Mvc/samples/MvcSandbox/Startup.cs [hunk 1]
```diff
@@ -66,13 +70,10 @@ public void Configure(IApplicationBuilder app)
 
                 builder.MapControllers();
                 builder.MapRazorPages();
-
-                builder.MapFallbackToController("Index", "Home");
+                builder.MapComponentHub<MvcSandbox.Components.App>("app");
```
Comment (javiercn) [inline src/Mvc/samples/MvcSandbox/Startup.cs]: I think you need to pass the path here too.

Comment (rynowak) [reply src/Mvc/samples/MvcSandbox/Startup.cs]: Path to what? This is working

Comment (javiercn) [reply src/Mvc/samples/MvcSandbox/Startup.cs]: Actually, nevermind. This should work just fine.

**Review by javiercn — APPROVED** [review]

**Comment by rynowak** [comment]
@aspnet/brt

**Comment by rynowak** [comment]
still fails after rebasing

**Comment by javiercn** [comment]
The agent is busted. Needs to be reboot.

**Comment by anurse** [comment]
Yeah, this is the file locking that's plaguing all our agents.

**Pull request merged**